### DISCUSSION TITLE
[Bug] Preview checkout customization

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#390] [Help] [answered]
The preview in the checkout customization editor is not shown anymore. There is a 404 - not found error.

**Comment by SusannaSwell** [comment]
hello @josias-hosang I was able to reproduce the issue, but only in test store mode.
are you experiencing this in test or live store mode?

**Reply by logeshswell** [reply]
I think this issue could be applicable only when there is no carts in the corresponding env.

**Comment by josias-hosang** [comment]
Ahoi @SusannaSwell, yesterday it was only in test mode, yes. But today I'm not able to preview also in live mode. @logeshswell I created a cart now (test mode), but still same not-found message.

**Reply by logeshswell** [reply]
Yes, it occurs for me too in test env. @SusannaSwell Can you file a bug ticket for it?
<img width="1512" alt="Screenshot 2023-09-27 at 8 01 11 PM" src="https://github.com/swellstores/community/assets/100766131/0e48ed63-2ba1-45bc-b775-f4460464f7d8">

**Reply by logeshswell** [reply]
It looks like the hosted checkout makes the checkout API call always to the live env instead of test env.

**Reply by steven-k1** [reply, marked as answer]
@josias-hosang A fix has been released for this. The checkout API calls should now correctly route through the proper environment.
